queue_services/entity-emailer/src/entity_emailer/config.py

```diff
@@ -97,6 +97,23 @@ class _Config():  # pylint: disable=too-few-public-methods
         name=DB_NAME,
     )
 
+    TRACKER_DB_USER = os.getenv('TRACKER_DATABASE_USERNAME', '')
+    TRACKER_DB_PASSWORD = os.getenv('TRACKER_DATABASE_PASSWORD', '')
+    TRACKER_DB_NAME = os.getenv('TRACKER_DATABASE_NAME', '')
+    TRACKER_DB_HOST = os.getenv('TRACKER_DATABASE_HOST', '')
+    TRACKER_DB_PORT = os.getenv('TRACKER_DATABASE_PORT', '5432')
+    TRACKER_SQLALCHEMY_DATABASE_URI = 'postgresql://{user}:{password}@{host}:{port}/{name}'.format(
+        user=TRACKER_DB_USER,
+        password=TRACKER_DB_PASSWORD,
+        host=TRACKER_DB_HOST,
+        port=int(TRACKER_DB_PORT),
+        name=TRACKER_DB_NAME,
+    )
+
+    SQLALCHEMY_BINDS = {
```

Comment:
This was required in order to support binding to lear db and tracker db.  Model classes for tracker db require a ` __bind_key__ = 'tracker'` in order to support multiple db binds.